.github/workflows/CI.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+on: [pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [ 18.x, 20.x ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Install Dependencies
+        run: npm install
+      - name: Run Build
+        run: npm run build
+      - name: Lint Codebase
+        run: npm run lint
+      - name: Unit Tests with Coverage
+        run: npm run test:cov
```

Comment:
Should we have it post test coverage in the PR?

Like so:

https://github.com/algorandfoundation/liquid-auth-use-wallet-client/blob/63eafc5477014aaf8c056c546f3ff076c51985b4/.github/workflows/test.yaml#L33

https://github.com/algorandfoundation/liquid-auth-use-wallet-client/pull/1

Reply:
That's a good idea, we may want to use a service like Codecov or Codeclimate that supports multiple frameworks. May be out of scope for this release

Reply:
Now that we have Codecov, do you want to set it up with this? Or should we have a new ticket to do that

Reply:
Would label it a `wont-fix` for now.